avm/res/dev-test-lab/lab/artifactsource/main.json

```diff
@@ -0,0 +1,151 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+  "languageVersion": "2.0",
+  "contentVersion": "1.0.0.0",
+  "metadata": {
+    "_generator": {
+      "name": "bicep",
+      "version": "0.25.53.49325",
```

Comment:
Placeholder to merge after publish `--with-source` is enabled

Reply:
ping @eriqua (just to force a notification)